### ISSUE TITLE
Use of aria-live inconsistent across www.gov.uk search results

### ISSUE THREAD

**Issue by tombye** [alphagov/finder-frontend#612]
Pages that display search results with filters in a left-hand column do not identify the summary information consistently to assistive tech'.

## Overview

When JS is enabled, the search results on these pages are updated by AJAX.

The change in the page content needs to be announced to users of assistive tech'. This is done by setting `aria-live` on the results summary paragraph but the value it is set to is not the same across www.gov.uk.

The implementation of the live-region is also slightly different for the search page compared to the policies page.

## What needs to be done

One value should be chosen for the `aria-live` attribute and make it the same everywhere.

The implementation is also slightly different on the search page. Both pages should be made the same.

Note: work done on this a while ago was documented in [this excellent blog post](https://gdstechnology.blog.gov.uk/2014/08/14/improving-accessibility-on-gov-uk-search/).

## Pages owned by finder-frontend

The pages I've found of this type (there may be others) are:

1. https://www.gov.uk/government/publications
2. https://www.gov.uk/government/consultations
3. https://www.gov.uk/government/statistics
4. https://www.gov.uk/government/announcements
5. https://www.gov.uk/government/policies
6. https://www.gov.uk/search

5 & 6 are owned by this app. I've raised [another issue for this on 1-4](https://github.com/alphagov/whitehall/issues/4383).

**Comment by selfthinker** [comment]
I looked at all the linked pages but can only see one value (`aria-live="assertive"`). Has this been fixed? Only https://www.gov.uk/government/statistics doesn't have any `aria-live` at all. (At least I couldn't find it.)

But having briefly looked at the implementation now, I would improve two other things:

1. What is wrapped in the div with the `aria-live` includes the number of results (in an h2), the facet tags and on some pages the "get email alerts" and "subscribe to feed" links. At least those links should not be announced to screen readers because they don’t change. (Actually, I assume the URL changes, but that won't be apparent to anyone, including screen reader users.) The facet tags can be up for debate if it makes sense for them to be announced, but I would suggest to only wrap the h2 (or add the `aria-live` directly to the h2).

2. After filtering at least once, the h2 gets nested in another h2. That is invalid HTML.

**Comment by kr8n3r** [comment]
@injms or @andysellick 
I have point 2 from ⬆️ as a ticket but it seems that is has been fixed.
can either of you confirm or alternatively show how to replicate (in case i'm missing something)?

**Comment by andysellick** [comment]
@kr8n3r yeah I think I remember spotting that and fixing it.

**Comment by kr8n3r** [comment]
thanks Andy

**Comment by andysellick** [comment]
@tombye I believe we replaced all of the finders that this ticket is talking about (originally from whitehall, now rebuilt in finder-frontend) and the aria issue should have now been fixed. Can you confirm if this is the case?

**Comment by tombye** [comment]
Looks like all of these now share the same implementation and the issues @selfthinker raised are now fixed so I'm marking this as resolved 👍
